src/context-providers/display/display-provider.tsx

```diff
@@ -0,0 +1,63 @@
+import { noop } from "lodash";
```

Comment:
use sub-directory import to reduce import size

```suggestion
import noop from "lodash/noop";
```

Reply:
changes made in the last push